docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -1,15 +1,16 @@
 # Permit Device To Install Applications
 
```

Comment:
Suggestion:
Request you to provide an introductory line for this section.

Reply:
We realized that this article "permit-device-to-install-apps.md" is actually for older Galaxy Watch series that don't support .NET platform. We should remove it from the documentation.

Reply:
@hakkyu-samsung
Thank you for the explanation.
I will do as suggested.